### PR TITLE
Friendship Milestone 5

### PR DESCRIPTION
**Pull request by EricMbouwe**
### Added 5 Rspec test files in spec folder

  - comments_spec.rb  
  - friendships_spec.rb  
  - menu_spec.rb  
  - posts_spec.rb  
  - users_spec.rb

### CODE REVIEW

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi Team      
#### Great Job so far, but there are a few more things required. 
### General
 - [x]  The project should have a descriptive README.md 
* [x] User can invite friends 
* [x] User can see friendships requests
* [x] User has to accept friendship request to start friendship
* [x] User can reject a friendship request
* [x] There is a button to invite user to friendship on users list page and on single user page 
* [x] Posts timeline includes posts written by logged-in user and their friends
- [x] Lets write unit and integration tests .
### Make changes and submit another code review request.
*Happy coding* 
(Any questions Feel free to contact me on slack @kobiyoyo)

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,9 +1,71 @@
 <section class="user-section">
-  <h2><%= "Name: #{@user.name}" %></h2>
+  <h2>
+    <%= @user.name %> 
+    <% unless @user == current_user %>
```
Comment (wathigo) [inline app/views/users/show.html.erb:4]: **[REQUIRED]**
* Let's remove all the logic from the views. Views related logic should be place in the helpers. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright)
Resources:
[Where Do I Put My Code?](http://codefol.io/posts/Where-Do-I-Put-My-Code)
[The Beginner's Guide to Rails Helpers](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers)

**Review by wathigo — CHANGES_REQUESTED** [review]
### Changes Required
Great work on the friendship feature.
I like the way you have implemented friendship rejection.

**REQUIRED**
* Let's remove all the logic from the views. See the commeny :point_down: 

**After implementing the changes, submit another request for review.**

**Happy Coding :computer: 
Simon Wathigo.**

File: app/views/users/show.html.erb (hunk 1 above)
Comment (ioanniskousis) [reply app/views/users/show.html.erb:4]: The only 'if' statements are to decide the appearance of elements
There is no logic in the views
You did not understand our code probably

**Review by whiz25 — APPROVED** [review]
## Good Work :clap: 
You can merge and proceed to the next milestone.

### Martin Afani

**Pull request merged**